tests/integration/models/release.spec.js

```diff
@@ -276,6 +276,44 @@ describe('Release Model', function () {
 					});
 				});
 			});
+
+			describe('balena.model.release.setIsInvalidated()', function () {
+				releaseRetrievalFields.forEach((field) => {
+					parallel('', function () {
+						it(`should validate a release by ${field}`, async function () {
+							const invalidRelease = testReleaseByField[field];
+							await balena.models.release.setIsInvalidated(
+								invalidRelease[field],
+								false,
+							);
+							const validatedRelease = await balena.models.release.get(
+								invalidRelease.id,
```

Comment:
NIT: Add a `$select` here as per my earlier comment (and in the lower `release.get()` ) so that we don't waste resources in retrieving unnecessary properties.